src/sortStudents.ts

```diff
@@ -1,7 +1,94 @@
 // describe Student type
 // create and export SortType enum
-// create SortOrder type
+// create SortOrder type]
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+// import { FileWatcherEventKind } from "typescript";
+function averageValue(values: number[]): number {
+  return values.reduce(
+    (sum, a) => sum + a,
+    0,
+  );
+}
+
+interface Student {
+  name: string;
+  surname: string;
+  age: number,
+  married: boolean,
+  grades: number[],
+}
+
+export enum SortType {
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades'
+}
+
+enum SortOrder {
+  asc = 'asc',
+  desc = 'desc'
```

Comment:
```suggestion
  Ascending = 'asc',
  Descending = 'desc'
```
Give meaningful names